source/upgrade.txt

```diff
@@ -132,6 +132,12 @@ Version 4.0 Breaking Changes
   UUID values must explicitly specify which representation to use. You can
   specify the representation in the ``uuidRepresentation`` property of
   ``MongoClientSettings``.
+
+  The UUID representation that you specify strictly controls how the driver
+  decodes UUIDs. For example, if you specify any of the ``LEGACY`` UUID
+  representations, the driver decodes only binary objects of subtype 3
+  as UUIDs. If you specify the ``STANDARD`` UUID representation, the
+  driver decodes only binary objects of subtype 4 as UUIDs.
```

Comment:
Suggestion: 
While I read that it "strictly controls how the driver decodes UUIDs", and it may currently be implied, I think is important to explicitly call out that the prior version disregarded the setting. Perhaps something like:

"Previous versions disregarded the value of this setting setting when decoding UUIDs."

Also, I think it could be helpful to mention what would happen if they had been using the wrong encoding. Does it throw an exception and if so, what type? Users searching for the symptom could identify the issue more easily and correct the problem.

Reply:
My testing may have been incomplete, but I wasn't able to get an exception thrown. Leaving as is for now.